### PR TITLE
[Testing] Enabling ported UITests from Xamarin.UITests to Appium - 23

### PR DESCRIPTION
**Pull request by ghost**
### Description of Change

This PR focuses on re-enabling and updating following tests from the Xamarin.UITest framework to be compatible with Appium. The tests, previously commented out, are reviewed, and modified to ensure they are functional with the Appium framework. We are going to enable tests in blocks in different PRs. This is the 23rd group of tests enabled.

Also include the helper methods `ShowFlyout` with tap and swipe actions. 

### TestCases

- **FlyoutBehaviorShell**

### CODE REVIEW

**Review by jsuarezruiz — DISMISSED** [review]

**Review by rmarinho — DISMISSED** [review]

**Comment by anandhan-rajagopal** [comment]
Resolved the conflicts.

File: src/TestUtils/src/UITest.Appium/HelperExtensions.cs [hunk 1]
```diff
@@ -1779,41 +1779,247 @@ public static IDictionary<string, object> GetSystemBars(this IApp app)
...
+
+		/// <summary>
+		/// Taps an item in the FlyoutPage flyout menu.
+		/// </summary>
+		/// <param name="app">The IApp instance representing the application.</param>
+		/// <param name="flyoutItem">The text or accessibility identifier of the flyout item to tap.</param>
+		public static void TapInFlyoutPageFlyout(this IApp app, string flyoutItem)
+		{
+			app.TapInFlyout(flyoutItem, false);
+    }
```
[the remark this answers is not in the document]
Comment (anandhan-rajagopal) [reply src/TestUtils/src/UITest.Appium/HelperExtensions.cs]: @MartyIX, Committed the suggested changes, kindly review and share your concerns

**Review by jfversluis — CHANGES_REQUESTED** [review]
I think I missed something with the merge, could you please check and rebase? Thanks!

**Review by jfversluis — APPROVED** [review]

**Pull request merged**